overlord/ifacestate/handlers.go

```diff
@@ -84,43 +84,6 @@ func (m *InterfaceManager) doSetupProfiles(task *state.Task, tomb *tomb.Tomb) er
 		return err
 	}
 
-	var corePhase2 bool
```

Comment:
on the upgrade the old snapd will still have created the 2nd setup, so we still need to for a while to check for the flag and turn the 2nd setup into a noop

Reply:
it's a setup-profiles (with a flag), it's not unknown